libcodechecker/cmd/cmd_line_client.py

```diff
@@ -474,11 +544,7 @@ def register_client_command_line(argument_parser):
     listruns_parser = subparsers.add_parser('runs',
                                             formatter_class=ADHFormatter,
                                             help='Get the run data.')
-    listruns_parser.add_argument('--host', type=str, dest="host",
-                                 default='localhost',
-                                 help='Server host.')
-    listruns_parser.add_argument('-p', '--port', type=str, dest="port",
-                                 default="8001", help='HTTP Server port.')
+    add_server_arguments(listruns_parser)
```

Comment:
Please revert this hunk change, and all such, related to `add_server_arguments` below.